amf-cli/shared/src/test/resources/validations/async20/validations/binding-wrong-key.yaml

```diff
@@ -0,0 +1,35 @@
+asyncapi: 2.5.0
+info:
+  title: API with GooglePubSub Message Binding Validation
+  version: 1.0.0
+servers:
+  googlePubSubServer:
+    url: localhost:9092
+    protocol: googlepubsub
+    description: A Google Pub/Sub server for messaging.
+    extraKey: an extra key
```

Comment:
Not necessary to test this. Server node is not new.

Reply:
deleted